scala-package/deploy/pom.xml

```diff
@@ -0,0 +1,137 @@
+    <profile>
+      <id>release</id>
+      <properties>
+        <revision>1.5.0</revision>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.5</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>ossrh</serverId>
+              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>false</autoReleaseAfterClose>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <attach>false</attach>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>com.google.code.maven-replacer-plugin</groupId>
+        <artifactId>replacer</artifactId>
+        <version>1.5.3</version>
+        <executions>
+          <execution>
+            <phase>deploy</phase>
+            <goals>
+              <goal>replace</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <file>${basedir}/src/main/deploy/deploy.xml</file>
+          <outputFile>${project.build.directory}/deploy.xml</outputFile>
+          <replacements>
+            <replacement>
+              <token>DESCRIPTION</token>
+              <value>${project.description}</value>
+            </replacement>
+            <replacement>
+              <token>ARTIFACT_ID</token>
+              <value>${ARTIFACT_ID}</value>
+            </replacement>
+            <replacement>
+              <token>PROJECT_VERSION</token>
+              <value>${project.version}</value>
+            </replacement>
+            <replacement>
+              <token>SCALA_VERSION</token>
+              <value>${scala.version}</value>
+            </replacement>
+          </replacements>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>deploy-file</id>
+            <phase>deploy</phase>
+            <goals>
+              <goal>deploy-file</goal>
+            </goals>
+            <configuration>
+              <description>${project.description}</description>
+              <url>file://${project.build.directory}/repo</url>
+              <groupId>${project.groupId}</groupId>
+              <artifactId>${ARTIFACT_ID}</artifactId>
+              <version>${project.version}</version>
+              <packaging>jar</packaging>
+              <pomFile>${project.build.directory}/deploy.xml</pomFile>
+              <file>${rootdir}/assembly/target/mxnet-full_2.11-INTERNAL.jar</file>
```

Comment:
The Internal placeholder will be replaced by the `linux/osx` and `cpu/gpu` right?

Reply:
the real artificatId that will be published to nexus is in the deploy.xml